Issues building from source

### The First Issue

Because of 5b7d9f84364b459f07fb1f43aaa2cccc9af4bbb2 , the build fails on my machine. It would be useful to work on the project without adding these to `gradle.properties`.

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/anandbeh/git/DazzleConf/xtomlj/build.gradle' line: 283

* What went wrong:
A problem occurred evaluating root project 'xtomlj'.
> Could not get unknown property 'ossrhUsername' for Credentials [username: null] of type org.gradle.internal.credentials.DefaultPasswordCredentials_Decorated.
```

Adding ossrhUsername and ossrhPassword works around this issue.

### The Second Issue

SpotlessJava fails because of formatting violations in TomlSerializer. I've made a PR for this.

### Longer-Term Solution
As an easy CI solution, you can use Github actions to run the gradle build. That way PRs and commits can be made sure to build correctly.

--- a/build.gradle
+++ b/build.gradle
@@ -277,13 +277,15 @@ publishing {
       }
     }
   }
-  repositories {
-    maven {
-      credentials {
-        username = "$ossrhUsername"
-        password = "$ossrhPassword"
+  if (System.getenv('MAVEN_CENTRAL_DEPLOY') == 'true') {
+    repositories {
+      maven {
+        credentials {
+          username = "$ossrhUsername"
+          password = "$ossrhPassword"
+        }
+        url "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
       }
-      url "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
     }
   }
 }

